versions/3.0.md

```diff
@@ -191,6 +192,30 @@ Contact information for the exposed API.
 
 Field Name | Type | Description
 ---|:---:|---
+<a name="languagesCurrent"></a>current | `string` | The language code for the current document.
+<a name="supportedLanguages"></a>supportedLanguages | [Parameter Object] | A parameter object of type `in: header` or `in: query` used to describe how to fetch the OAS document in the language allowed by the `enum`
+
+Example:
+```yaml
+languages:
+  current: en-us
+  supportedLanguages:
+    in: query
+    name: lang
+    type: string
+    enum:
+      - en-us
+      - es
```

Comment:
many use the header, so you can't describe it like such.  I'll see if we can use a filename optionally.